src/lib/ngsi/ContextAttribute.cpp

```diff
@@ -85,7 +85,7 @@ ContextAttribute::ContextAttribute(ContextAttribute* caP)
   numberValue           = caP->numberValue;
   boolValue             = caP->boolValue;
   compoundValueP        = caP->compoundValueP;
-  caP->compoundValueP   = NULL;
+  caP->compoundValueP   = NULL;   // FIXME P10: really?? shouldn't it be a clone?
```

Comment:
I need a second opinion here... 

This seems to be a bug, as the constructor shouldn't modify the "source" being copied. In addition, a clone() for portencial compounds is missing, i.e.:

```
compoundValueP = caP->compoundValueP == NULL ? NULL : caP->compoundValueP->clone();
```